Documentation/AndroidInstallation.md

```diff
@@ -56,15 +56,39 @@ import com.oney.WebRTCModule.WebRTCModulePackage;  // <--- Add this line
      .setUseDeveloperSupport(BuildConfig.DEBUG)
 ```
 
-5.) Enable Java 8 support in your project. In `anroid/app/build.gradle` add:
+5.) Enable Java 8 support in your project. You will probably need to have React Native 0.55+ for this.
 
+5.a.) In `anroid/app/build.gradle` add:
 ```gradle
 compileOptions {
```

Comment:
Apparently, we need to add this configuration option in a section - otherwise, add them in android/app/build.gradle results in:

Could not find method compileOptions()

Those configuration options must be in the android part, right?
Just after defaultConfig?

Reply:
@k0pernicus That's right it should be inside `android` section. Will add that clarification

Reply:
Thanks 👍